tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_InferredAssetSQLExample.ipynb

```diff
@@ -0,0 +1,1215 @@
+              "}"
+            ]
+          },
+          "execution_count": 20,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "validator.expect_column_median_to_be_between(column=\"trip_distance\", min_value=0, max_value=1)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "f18d096f",
+      "metadata": {},
+      "source": [
+        "The observed value for our `yellow_tripdata_sample_2020_01` table where `vendor_id` = `2`  is going to be `1.6`, which means the Expectation fails"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "9c5016d8",
+      "metadata": {},
+      "source": [
+        "Now we run the same expectation again, but this time with `auto=True`. This means the `median` values are going to calculated across the `batch_list` associated with the `Validator` (ie 3 Batches for `yellow_tripdata_sample_2020_01`), which gives the min value of `1.5` and the max value of `5.23`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 21,
+      "id": "cdd821c3",
+      "metadata": {},
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "\n"
+          ]
+        },
+        {
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "24cdf5939eee463cb24c7a87dd3e10a9",
+              "version_major": 2,
+              "version_minor": 0
+            },
+            "text/plain": [
+              "Generating Expectations:   0%|          | 0/1 [00:00<?, ?it/s]"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "",
+              "version_major": 2,
+              "version_minor": 0
+            },
+            "text/plain": [
+              "Profiling Dataset:         0%|          | 0/1 [00:00<?, ?it/s]"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "c3b804d1c7774992aefe5c5bc747cbd9",
+              "version_major": 2,
+              "version_minor": 0
+            },
+            "text/plain": [
+              "Calculating Metrics:   0%|          | 0/9 [00:00<?, ?it/s]"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/plain": [
+              "{\n",
+              "  \"success\": true,\n",
+              "  \"expectation_config\": {\n",
+              "    \"kwargs\": {\n",
+              "      \"column\": \"trip_distance\",\n",
+              "      \"min_value\": 1.6,\n",
+              "      \"max_value\": 1.6,\n",
```

Comment:
This is strange - I would have thought the expectation would now be using the min_value of 1.5 and max_value of 5.23 computed from the auto run (not 1.6)

Reply:
Might be a bug. Will look into this :)

Reply:
Found it. I wasn't using the full `batch_list`... fixed and updated